mr/src/test/java/org/apache/iceberg/mr/TestIcebergInputFormats.java

```diff
@@ -52,8 +50,7 @@
 import org.apache.iceberg.expressions.Expressions;
 import org.apache.iceberg.hadoop.HadoopCatalog;
 import org.apache.iceberg.hadoop.HadoopTables;
-import org.apache.iceberg.mr.mapred.Container;
-import org.apache.iceberg.mr.mapred.MapredIcebergInputFormat;
+import org.apache.iceberg.mr.mapred.TestMapredIcebergInputFormat;
```

Comment:
Do we still need this class / import?

Reply:
I still have issue with testResiduals test on hive3, i will try to run test if i force desactivated vectorization on it.

Reply:
hi @pvary,
I try to avoid this change but i have lot of issue to make org.apache.iceberg.mr.TestIcebergInputFormats.testFailedResidualFiltering and org.apache.iceberg.mr.TestIcebergInputFormats.testResiduals work for hive3
Either I desactivate these two tests for hive3 or keep the current implementation

Reply:
Do you still have the problems if you revert all of the changes of this file, and remove the new `TestMapredIcebergInputFormat` class? I do not see the reason behind this.

Reply:
Yes.
The reason it's the change in getSplits which now enforce SKIP_RESIDUAL_FILTERING using skipResidualFiltering.

Reply:
Isn't it an intended change?
If that breaks some test classes then we have to check if it is ok, or we missed some important cases.

Reply:
Yes it's intended change.
it's breaking test because the test test the behaviour of skip residual and memory model behaviour but for hive3 this behaviour with the change it's enforce by the vectorization.

Reply:
Then we should fix the test 😎

Reply:
I push the latest version of the tests.
For hive3, skip residual tests, I have the same issue on the mock so i switch to TestHiveIcebergStorageHandlerWithEngine.
And these tests are redundant because when i remove it it's break lot of others test in hive3.